src/commands/navigation.rs

```diff
@@ -79,15 +73,33 @@ fn advance(
             .children(CommitSet::from(current_oid))?
             .difference(&dag.obsolete_commits);
         let children = sort_commit_set(repo, dag, &children)?;
+
+        let children_pluralize = Pluralize {
+            amount: i,
+            plural: "children",
+            singular: "child",
+        };
         let header = format!(
-            "Found multiple possible next commits to go to after traversing {} children:",
-            i
+            "Found multiple possible next commits to go to after traversing {}:",
+            children_pluralize.to_string(),
         );
 
         current_oid = match (towards, children.as_slice()) {
             (_, []) => {
-                // It would also make sense to issue an error here, rather than
-                // silently stop going forward commits.
+                writeln!(
+                    effects.get_output_stream(),
+                    "{}",
+                    printable_styled_string(
+                        glyphs,
+                        StyledString::styled(
+                            format!(
+                                "No more child commits to go to after traversing {}.",
```

Comment:
This is just a warning, not an error, right? Does it still make sense to use red for this message? I would expect yellow.